terminus/builders/simple_city_builder.py

```diff
@@ -62,9 +63,10 @@ def _create_surrounding_ring_road(self, city, size):
         city.add_road(ring_road_4)
 
     def _create_blocks(self, city, size):
+        blockBuilder = SquareBlockBuilder()
         for x in range(size):
             for y in range(size):
-                block = Block(Point(x*100+50, y*100+50))
+                block = blockBuilder.gen_block(Point(x*100+50, y*100+50, 0), 95)
```

Comment:
Could you move this to a class method instead of having a separate builder? So we can do `Block.square(Point(x*100+50, y*100+50), 95)`.